agent/bench-scripts/postprocess/user-benchmark-wrapper

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+import sys, json, os, logging, shutil
+
+benchmark_run_dir = sys.argv[1]
+benchmark_duration = int(sys.argv[2])
+
+with open('%s/user-benchmark-summary.json' % benchmark_run_dir, 'a+') as data_file:
+    try:
+        data = json.load(data_file)
+    except ValueError as e:
+        print e
+        position = data_file.tell()
+        logging.exception("current position: %d", position)
```

Comment:
Consider only logging the exception if the position is not 0.  If the position is 0, it does not seem like we need to log anything since we log a warning to that effect on line 24.